sympy/printing/latex.py

```diff
@@ -87,10 +87,10 @@
     'scr': lambda s: r'\mathscr{'+s+r'}',
     'frak': lambda s: r'\mathfrak{'+s+r'}',
     # Brackets
-    'norm': lambda s: r'\left\lVert{'+s+r'}\right\rVert',
+    'norm': lambda s: r'\left\|{'+s+r'}\right\|',
     'avg': lambda s: r'\left\langle{'+s+r'}\right\rangle',
-    'abs': lambda s: r'\left\lvert{'+s+r'}\right\rvert',
-    'mag': lambda s: r'\left\lvert{'+s+r'}\right\rvert',
+    'abs': lambda s: r'\left\|{'+s+r'}\right\|',
+    'mag': lambda s: r'\left\|{'+s+r'}\right\|',
```

Comment:
these too look wrong, isn't `\|` two bars?   like a norm?  You can't replace both `\lVert` and `\lvert` with the same thing!